src/main/java/uk/gov/hmcts/juror/api/moj/utils/JurorResponseUtils.java

```diff
@@ -61,6 +69,22 @@ public static PaperResponse createMinimalPaperSummonsRecord(Juror juror,
         return jurorPaperResponse;
     }
 
+    public static void updateCurrentOwnerInResponseDto(JurorPoolRepository jurorPoolRepository,
+                                                       IJurorResponse responseDto) {
+
+        // set the current owner.  Need to ensure the current owner is returned as the owner can change if, for
+        // example, the juror is transferred to a different pool
+        List<JurorPool> jurorPools =
+            JurorPoolUtils.getActiveJurorPoolRecords(jurorPoolRepository, responseDto.getJurorNumber());
+
```

Comment:
I believe an account should only have one active JurorPool at a time?

(Please check with Craig)